src/pages/index.js

```diff
@@ -1,60 +1,64 @@
+import get from "lodash/get"
+import { Link, graphql } from "gatsby"
 import React from "react"
-import Link from 'gatsby-link'
-import styled from 'styled-components'
-import get from 'lodash/get'
+import styled from "styled-components"
 
-import BlogPost from '../components/BlogPost.js'
-import gridPaper from '../images/grid-paper.png'
+import blogPostBorder from "../images/blog_post_border.svg"
+import dotGridPaper from "../images/dot-grid.png"
+import FeaturedBlogPost from "../components/FeaturedBlogPost"
 
 const MainWrapper = styled.div`
-  background-color: #a2e9f9;
-  border-top: 1px solid #7cc2fd;
-  padding: 0 20px;
+  background-image: url(${dotGridPaper});
+  background-repeat: repeat;
+  display: flex;
+  height: 700px;
+  width: 1400px;
+`
+
+const AboutWrapper = styled.section`
+  align-self: flex-end;
+  margin: 0 75px 200px;
 `
 
 const SiteTitle = styled.h1`
+  font-family: "Fredericka the Great";
   color: #f3c39a;
 `
 
-const FeaturedPostWrapper = styled.div`
-  background-color: #eee;
-  border: 2px solid #ccc;
-  padding: 0 20px;
-`
+const style = {
+  textDecoration: "none",
+  color: "#f268ae",
+}
 
-const GridPaperImageWrapper = styled.div`
-  background-color: #fff;
-  background-image: url(${gridPaper});
-  background-repeat: repeat;
-  background-size: cover;
-  border-top: 1px solid #f4f4f4;
-  display: none;
-  grid-column: 1 / 2;
-  grid-row: 2 / 3;
-  width: 100%;
-  @media screen and (min-width: 40rem) {
-    display: block;
-  }
+const FeaturedPostWrapper = styled.section`
+  align-self: flex-end;
+  background-image: url(${blogPostBorder});
+  background-repeat: none;
+  width: 477px;
+  height: 425px;
+  padding: 10px;
+  margin: auto;
 `
 
 class BlogIndex extends React.Component {
   render() {
-    const post = get(this, 'props.data.featured.edges[0].node')
+    const post = get(this, "props.data.featured.edges[0].node")
 
     return (
       <MainWrapper>
-        <SiteTitle>
-          Anxious in Brooklyn
-        </SiteTitle>
-        <GridPaperImageWrapper>
+        <AboutWrapper>
+          <SiteTitle>Anxious in Brooklyn</SiteTitle>
+          <Link to={`/about/`} style={style}>
+            About this blog
+          </Link>
+          <br />
+          <Link to={`/blog-posts/`} style={style}>
+            All Posts
+          </Link>
+        </AboutWrapper>
         <FeaturedPostWrapper>
-          <BlogPost
-            key={post.slug}
-            post={post}
-          />
+          <FeaturedBlogPost key={post.slug} post={post} />
```

Comment:
no need for a `key` prop here. Key is used by React to determine changes in state when you are iterating over an array:

```jsx
['foo', 'bar', baz'].map((str) => <div key={str}>{str}</div> 
```
`key` prop here tells react when it needs to re-render. 

In this case `<FeaturedBlogPost />` component is static and it "exists" in React's virtual dom by default.

Let me know if this makes sense. Happy to explain this in person

Reply:
Missed during refactoring.